cluster/libexec/wwinit/50-ntpd.init

```diff
@@ -20,9 +20,14 @@ fi
 
 wwreqroot
 
+if  rpm -q chrony >/dev/null 2>&1 ; then
```

Comment:
@jmstover Thoughts on if we should bail from 50-ntpd.init if chrony is installed? It is the default in EL7 and 8, but at least in my environment we still use ntpd everywhere. Alternatively we could reverse the order, and check for ntp.conf, if not found and chrony *is* found, exit 0 cleanly. If ntp.conf isn't found *and* chrony is not found then we exit 1.

Reply:
Sounds fine to me. Admins shouldn't have both installed at the same time, but I just confirmed you can. Can you confirm these changes will do what you're asking? Changing the filenames so there's a clear order to wwinit.

1. 51-chrony.init: if ntp or openntpd found then return error 0.
2. 51-chrony.init: if chrony not found, return error 1.
3. 52-ntp.init: if ntp/openntpd not found, check for chrony.
4. 52-ntp.init: if chrony found, return error 0, else return error 1.